include/big_mpi_compat.h

```diff
@@ -314,6 +314,75 @@ namespace BigMPICompat
     return MPI_SUCCESS;
   }
 
+
+
+
+
+
+
+//My work: 
```

Comment:
please remove this line and make sure there are only ~3 empty lines between functions.